src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -160,7 +160,7 @@ class SaveInProgressIntro extends React.Component {
             </div>
           );
         }
-      } else if (prefillAvailable && !verifiedPrefillAlert) {
+      } else if (!prefillAvailable && !verifiedPrefillAlert) {
```

Comment:
Why are things changing here?

Reply:
@smeneni, this isn't the correct place to make these changes. In general, we should not be making changes to platform code.